source/simulator/freesurface.cc

```diff
@@ -78,6 +78,25 @@ namespace aspect
                         "can preserve the mesh quality better, but at the "
                         "cost of slightly poorer mass conservation of the "
                         "domain.");
+      prm.declare_entry ("Additional tangential mesh velocity boundary indicators", "",
+                         Patterns::List (Patterns::Anything()),
+                         "A comma separated list of names denoting those boundaries "
+                         "where there the mesh is allowed to move tangential to the "
+                         "boundary. All tangential mesh movements along "
+                         "those boundaries that have tangential material velocity "
+                         "boundary conditions are allowed by default, this parameters "
+                         "allows to generate mesh movements along other boundaries that are "
+                         "open, or have prescribed material velocities or tractions."
+                         "\n\n"
+                         "The names of the boundaries listed here can either by "
+                         "numeric numbers (in which case they correspond to the numerical "
```

Comment:
"numeric numbers" seems redundant. Perhaps digits? Though I see now that this is in all of the descriptions of the boundary parsing...